common/src/address.rs

```diff
@@ -142,6 +142,7 @@ impl ReservedRackSubnet {
 
 const SLED_AGENT_ADDRESS_INDEX: usize = 1;
 const SWITCH_ZONE_ADDRESS_INDEX: usize = 2;
+const NTP_ZONE_ADDRESS_INDEX: usize = 3;
```

Comment:
Yes, what you have described is as intended - once underlay is up, an NTP zone is started on it. On a scrimlet, a switch zone is also started. Any switch zone is ::2 on the sled subnet, the NTP zone is ::3.

I originally experimented with having the NTP zones on the bootstrap network. With the right chrony configuration it would allow them to synchronise together to achieve a consistent time, but I think that's unnecessary, and there are discovery challenges.

More context in my other replies here.